cmd/metal-api/internal/service/network-service.go

```diff
@@ -560,6 +561,12 @@ func (r *networkResource) freeNetwork(request *restful.Request, response *restfu
 	for _, prefix := range nw.Prefixes {
 		err = r.ipamer.ReleaseChildPrefix(ctx, prefix)
 		if err != nil {
+			var connectErr *connect.Error
+			if errors.As(err, &connectErr) {
```

Comment:
Probably we cannot skip here as then we have orphaned prefixes after freeing the network.